README.md

```diff
@@ -10,6 +10,7 @@ This action for [Changesets](https://github.com/atlassian/changesets) creates a
 - version - The command to update version, edit CHANGELOG, read and delete changesets. Default to `changeset version` if not provided
 - commit - The commit message to use. Default to `Version Packages`
 - title - The pull request title. Default to `Version Packages`
+- skipNpmrcCheck - Check if there is a user-defined `.npmrc` file with a proper npm registry and an `authToken`. If not, a default `.npmrc` file is created.
```

Comment:
@Andarist I added this option to opt-out of the .npmrc checks.

I find it useful in case I don't need the use the default npm registry (for example when using github packages).

What do you think?